### PR TITLE
Dataviz - Build the dataviz dock interface from the Drag&Drop HTML layout

### PR DESCRIPTION
**Pull request by mdouchin**
Lizmap plugin allows building a Drag & Drop HTML layout to organize the plots in tabs and groups like it is possible for the editing form (Drag & drop form layout).

This commit uses the new `dataviz_drag_drop` configuration stored in the JSON config file (*.cfg) and creates the corresponding HTML

**Example tree layout in Lizmap plugin**

image

**Result in Lizmap Web Client web interface**

image

**Funded by**: Direction Départementale des Territoires et de la Mer de l’Hérault (DDTM 34)

### CODE REVIEW

**Comment by mdouchin** [comment]
cc @Gustry @nboisteault

File: tests/qgis-projects/tests/dataviz.qgs.cfg [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-        "lizmap_plugin_version_str": "3.9.1-alpha",
-        "lizmap_plugin_version": 30901,
-        "lizmap_web_client_target_version": 30500,
-        "lizmap_web_client_target_status": "Stable",
+        "qgis_desktop_version": 32214,
+        "lizmap_plugin_version_str": "3.14.1-alpha",
+        "lizmap_plugin_version": 31401,
+        "lizmap_web_client_target_version": 30700,
+        "lizmap_web_client_target_status": "Dev",
+        "instance_target_url": "https://demo.snap.lizmap.com/lizmap_3_7/",
```
Comment (Gustry) [inline tests/qgis-projects/tests/dataviz.qgs.cfg:8]: Maybe a bug on my side or not, but I was wondering if it's normal you have some blue in this new panel D&D when looking at your screenshot in the PR description.

**Comment by nboisteault** [comment]
@mdouchin I think [this error](https://github.com/3liz/lizmap-web-client/actions/runs/5202089543/jobs/9383188396?pr=3716#step:19:636) comes in the console. It is an error I see randomly with the attribute table, I've never found how to fix it so you might tell Cypress to ignore it.

**Comment by nboisteault** [comment]
@mdouchin I do that for the error : https://github.com/3liz/lizmap-web-client/blob/master/tests/end2end/cypress/integration/feature_toolbar-ghaction.js#L200

File: tests/qgis-projects/tests/dataviz.qgs.cfg (hunk 1 above)
Comment (Gustry) [reply tests/qgis-projects/tests/dataviz.qgs.cfg:8]: OK, it was just for me if you were hitting a bug of mine, that's it

**Pull request merged**